invenio_communities/assets/semantic-ui/js/invenio_communities/members/invitations/InvitationResultItem.js

```diff
@@ -20,6 +20,23 @@ import { RequestActionController } from "@js/invenio_requests";
 const formattedTime = (expires_at) =>
   DateTime.fromISO(expires_at).setLocale(i18next.language).toRelative();
 
+export const dropdownRoleOptionsSerializer = (roles) =>
```

Comment:
I am not understanding why this func is used here and in the `<ActionDropdown` below: I would expect that it is defined in some common place between the Users and Groups tabs. To be discussed in IRL.